### DISCUSSION TITLE
Possibility for tagged releases/deployments

### DISCUSSION THREAD

**Discussion by cryptiklemur** [vercel/vercel#4805] [Ideas]
It would be nice to have the option to make production deploys off of tagged github releases.

Not sure how feasible this is, but it would be nice to be able to link "releases" in github, with releases on vercel

**Comment by paulogdm** [comment]
Hello,

At the moment, the team is not considering this feature. However, we will certainly consider it once the product is more mature.
Thanks for your feedback and do let us know if you have any further messages for us.
